patches/chrome-android-BUILD.gn.patch

```diff
@@ -19,15 +19,23 @@ index ac52a743f73b7109f57df8f677518ca717812ba7..ff92fd9e070e290566ebfb401afc3b3a
      "//chrome/android/webapk/libs/common:splash_resources",
      "//chrome/app:java_strings_grd",
      "//chrome/browser/ui/android/widget:ui_widget_java_resources",
-@@ -543,6 +544,7 @@ java_group("chrome_all_java") {
+@@ -276,6 +277,7 @@ android_library("chrome_java") {
+     "$google_play_services_package:google_play_services_gcm_java",
+     "$google_play_services_package:google_play_services_iid_java",
+     "$google_play_services_package:google_play_services_tasks_java",
++    "$google_play_services_package:google_play_services_safetynet_java",
```

Comment:
this should be `deps += [ "//brave/android:chrome_java" ]` and have a dep (or public dep if needed) for https://github.com/brave/brave-core/pull/3731/files#diff-9fe2af9787818c4ff80aff1cbfae65f6R10